### ISSUE TITLE
I can not remove first version of Ruffle

### ISSUE THREAD

**Issue by cmlad26** [ruffle-rs/ruffle#3503]
After the start of chrome when i go to an application that uses ruffle the browser load the older version of ruffle.
![image](https://user-images.githubusercontent.com/76111979/109616550-f47d3580-7b3d-11eb-91bb-203f127b0e68.png)
I tried to download the latest version of browser extension and installed it in developer mode but the issue stil remains. (the capture has been made after i installed the extension from today 02.02.2021)

**Comment by Poopooracoocoo** [comment]
Just making sure: You did uninstall the extension first before installing the new version, right? 

And I'm assuming you meant version 2021-03-02, and that you're on Chrome 88 on Windows 10?

It could be a cache issue or something...

**Comment by cmlad26** [comment]
Yep was uninstalled and the corresponding folder has been removed.
It is the same issue for version 2021-03-01 and also for 2021-03-02, i'm using Windows 7 on one machine and windows 10 on other machine the problem was the same.
    On Tuesday, March 2, 2021, 02:33:04 PM GMT+2, Poopooracoocoo <notifications@github.com> wrote:  
 
 


Just making sure: You did uninstall the extension first before installing the new version, right?

And I'm assuming you meant version 2021-03-02, and that you're on Chrome 88 on Windows 10?

It could be a cache issue or something...

**Comment by Lord-McSweeney** [comment]
This issue is probably occurring when the site's outdated version of Ruffle loads before the Ruffle extension.
